commands.c

```diff
@@ -215,7 +212,7 @@ int mutt_display_message(struct Mailbox *m, struct Email *e)
       if (e->security & APPLICATION_SMIME)
         crypt_smime_getkeys(e->env);
       if (!crypt_valid_passphrase(e->security))
-        return 0;
+        goto cleanup;
```

Comment:
Will jumping to `cleanup` cause an issue since `cleanup` is defined as:

```C
cleanup:
  mutt_buffer_pool_release(&tempfile);
  return rc;
```

and `tempfile` does not exist when the `goto` is called?

Reply:
Argh!  UB!  Good catch.

Reply:
Fixed the commit, in-place